README.md

```diff
@@ -35,18 +35,9 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
     * Installed in `/etc/fuseki`
     * Dataset Path name `/test`
     * Persistent storage `/etc/fuseki/databases/test\_data`
-  * [Fcrepo-camel-toolbox 4.x](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox)
+  * [Fcrepo-camel-toolbox 5.x](https://github.com/fcrepo4-exts/fcrepo-camel-toolbox)
     * Installed in karaf
 
-### Fedora Configuration
-
-* By default, WebAC authorization is enabled on this Fedora installation.
-  * Three Fedora user accounts are available:
-    * user account `testuser`, with password `password1`
-    * user account `adminuser`, with password `password2`
-    * admin account `fedoraAdmin` with the password `secret3`
```

Comment:
Should we not leave the usernames and passwords here for users? We can remove the stuff below about disabling access control though.

Reply:
good point - I'll put the username/password info back in.

Reply:
👍